.evergreen/setup-gcp-instance.sh

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env bash
+
+set -o errexit
+if [ -z ${testgcpkms_key_file+omitted} ]; then echo "testgcpkms_key_file is unset" && exit 1; fi
+
+echo "${testgcpkms_key_file}" > ./testgcpkms_key_file.json
+export GCPKMS_KEYFILE=./testgcpkms_key_file.json
+
+"$GCPKMS_DRIVERS_TOOLS/.evergreen/csfle/gcpkms/create-and-setup-instance.sh"
```

Comment:
Perhaps dumb question – does this task also start a server? And is that server always a standalone?

Reply:
Yep, not dumb, this line: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/csfle/gcpkms/remote-scripts/setup-gce-instance.sh#L21